### PR TITLE
Initial functionality for S3 and local file writing for bulk fhir + hooked up anonymizer

### PR DESCRIPTION
**Pull request by bschofield-va**

### CODE REVIEW

File: anonymizer/src/main/java/gov/va/api/health/bulkfhir/anonymizer/patient/PatientAnonymizer.java [hunk 1]
```diff
@@ -68,6 +68,6 @@ boolean sanitizeMultipleBirthBoolean(Boolean multipleBirthBoolean, Integer multi
     if (multipleBirthInteger != null) {
       return multipleBirthInteger > 0;
     }
-    return multipleBirthBoolean;
+    return Boolean.TRUE.equals(multipleBirthBoolean);
```
Comment (bschofield-va) [inline anonymizer/src/main/java/gov/va/api/health/bulkfhir/anonymizer/patient/PatientAnonymizer.java]: good change

File: the-incredible-bulk/src/main/java/gov/va/api/health/bulkfhir/service/controller/publication/NonDistributedFileWorker.java [hunk 2]
```diff
@@ -24,20 +33,21 @@
...
 
   @Override
   @Async(PUBLICATION_BUILD_EXECUTOR)
   public CompletableFuture<FileBuildResponse> buildFile(FileClaim claim) {
     try {
-      fetchPatients(claim);
-      anonymizePatients();
-      writePatients();
+      List<Patient> patients = fetchPatients(claim);
+      writePatients(claim, patients.stream().map(patientAnonymizer()).map(jsonStringConverter()));
```
Comment (bschofield-va) [inline the-incredible-bulk/src/main/java/gov/va/api/health/bulkfhir/service/controller/publication/NonDistributedFileWorker.java]: Should we put a `filter(Objects:nonNull)` here just to make sure that if the JSON conversion failed for some unknown reason we don't emit the string _null_.

**Comment by bschofield-va** [comment]
FYI @dlennon-va
Just making sure you can see this PR

File: the-incredible-bulk/src/main/java/gov/va/api/health/bulkfhir/service/controller/publication/NonDistributedFileWorker.java (hunk 2 above)
Comment (dlennon-va) [reply the-incredible-bulk/src/main/java/gov/va/api/health/bulkfhir/service/controller/publication/NonDistributedFileWorker.java]: Yea, good call. Added that.

**Comment by bschofield-va** [comment]
lgtm

**Pull request merged**